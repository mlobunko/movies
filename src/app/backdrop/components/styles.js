import styled from 'styled-components';

export const StyledBackdropComponent = styled.div`
  min-height: calc(100vh - 3rem);
  max-height: 100vh;
  background: ${props =>
    props.backdropPath &&
    `
    linear-gradient(
      rgba(60, 60, 60, 0.8),
      rgba(60, 60, 60, 0.8)
    ),
    url(https://image.tmdb.org/t/p/original${
      props.backdropPath
    }) no-repeat center center / cover`};
`;

StyledBackdropComponent.displayName = 'StyledBackdropComponent';
