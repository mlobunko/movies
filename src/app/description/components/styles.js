import styled from 'styled-components';

export const StyledDescription = styled.div`
  width: 90%;
  max-width: 640px;
  margin: 0 auto;
`;

StyledDescription.displayName = 'StyledDescription';

export const StyledLabel = styled.span`
  color: rgb(184, 184, 184);
`;

StyledLabel.displayName = 'StyledLabel';

export const StyledList = styled.p`
  margin: 0;
`;

StyledList.displayName = 'StyledList';

export const StyledRow = styled.div`
  margin-top: 2rem;
  display: flex;
  justify-content: flex-start;
  text-align: left;
`;

StyledRow.displayName = 'StyledRow';

export const StyledRowLeft = styled.div`
  width: 7rem;
`;

StyledRow.displayName = 'StyledRowLeft';

export const StyledRowRight = styled.div`
  width: 7rem;
  margin-left: 4rem;
`;

StyledRowRight.displayName = 'StyledRowRight';

export const StyledTagline = styled.p`
  color: rgb(184, 184, 184);
`;

StyledTagline.displayName = 'StyledTagline';
