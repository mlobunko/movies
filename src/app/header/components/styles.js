import styled from "styled-components";

export const StyledHeader = styled.h1`
  margin: 0;
  text-align: center;
  height: 3rem;
  line-height: 3rem;
  vertical-align: center;
  color: rgb(223, 8, 19);
  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
`;

StyledHeader.displayName = "StyledHeader";
