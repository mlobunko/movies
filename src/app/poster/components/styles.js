import styled from 'styled-components';
import fallbackPoster from '../images/fallbackPoster.jpg';

export const StyledPoster = styled.div`
  position: absolute;
  width: 100%;
  height: 100%;
  background: ${props =>
    props.posterPath
      ? `url(https://image.tmdb.org/t/p/original${props.posterPath})
    no-repeat center center / cover`
      : `url(${fallbackPoster})
    no-repeat center center / cover`};
`;

StyledPoster.displayName = 'StyledPoster';

export const StyledPosterContainer = styled.div`
  width: 90%;
  max-width: 320px;
  margin: 0 auto;
`;

StyledPosterContainer.displayName = 'StyledPosterContainer';

export const StyledPosterContent = styled.div`
  position: relative;
  width: 100%;
  padding-bottom: 150%;
`;

StyledPosterContent.displayName = 'StyledPosterContent';
