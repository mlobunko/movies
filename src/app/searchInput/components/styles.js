import styled from 'styled-components';

export const theme = {
  container: {
    position: 'relative'
  },
  input: {
    width: '100%',
    background: 'rgba(0, 0, 0, 0)',
    color: 'white',
    height: 30,
    padding: '10px 20px',
    fontSize: 16,
    border: 'none',
    borderBottom: '1px solid white'
  },
  inputFocused: {
    outline: 'none'
  },
  inputOpen: {
    borderBottomLeftRadius: 0,
    borderBottomRightRadius: 0
  },
  suggestionsContainer: {
    display: 'none'
  },
  suggestionsContainerOpen: {
    display: 'block',
    position: 'absolute',
    width: '100%',
    background: 'rgba(0, 0, 0, 0.8)',
    borderBottomLeftRadius: 4,
    borderBottomRightRadius: 4,
    zIndex: 2
  },
  suggestionsList: {
    margin: 0,
    padding: 0,
    listStyleType: 'none'
  },
  suggestion: {
    cursor: 'pointer',
    padding: '10px 20px'
  },
  suggestionHighlighted: {
    backgroundColor: '#fff',
    color: '#000'
  }
};

export const StyledContainer = styled.div`
  width: 90%;
  max-width: 480px;
  margin: 0 auto;
  padding: 1rem 0rem 4rem 0rem;
`;
StyledContainer.displayName = 'StyledContainer';
