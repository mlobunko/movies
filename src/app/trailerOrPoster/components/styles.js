import styled from 'styled-components';

export const StyledPoster = styled.div`
  position: absolute;
  width: 100%;
  height: 100%;
  background: ${props => `url(https://image.tmdb.org/t/p/original${
    props.posterPath
  })
    no-repeat center center / cover`};
`;

StyledPoster.displayName = 'StyledPoster';

export const StyledTrailer = styled.iframe.attrs({
  src: props =>
    `https://www.youtube.com/embed/${props.trailerPath}?rel=0&amp;showinfo=0`,
  frameBorder: '0',
  allow: 'autoplay; encrypted-media',
  allowFullScreen: true
})`
  position: absolute;
  width: 100%;
  height: 100%;
`;

StyledTrailer.displayName = 'StyledTrailer';

export const StyledPosterContainer = styled.div`
  width: 90%;
  max-width: 320px;
  margin: 0 auto;
`;

StyledPosterContainer.displayName = 'StyledPosterContainer';

export const StyledTrailerContainer = styled.div`
  width: 90%;
  max-width: 640px;
  margin: 0 auto;
`;

StyledTrailerContainer.displayName = 'StyledTrailerContainer';

export const StyledPosterContent = styled.div`
  position: relative;
  width: 100%;
  padding-bottom: 150%;
`;

StyledPosterContent.displayName = 'StyledPosterContent';

export const StyledTrailerContent = styled.div`
  position: relative;
  width: 100%;
  padding-bottom: 56.25%;
`;

StyledTrailerContent.displayName = 'StyledTrailerContent';
