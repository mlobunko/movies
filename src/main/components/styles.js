import styled from "styled-components";

export const StyledContent = styled.div`
  min-height: calc(100vh - 3rem);
`;

export const StyledMainComponent = styled.div`
  background: rgb(0, 0, 0);
  color: rgb(255, 255, 255);
`;

StyledMainComponent.displayName = "StyledMainComponent";
