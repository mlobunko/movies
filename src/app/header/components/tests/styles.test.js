import React from 'react';
import { shallow } from 'enzyme';
import 'enzyme-to-json';

import { StyledHeader } from '../styles';

describe('StyledHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
