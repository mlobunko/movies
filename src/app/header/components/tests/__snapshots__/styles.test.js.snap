// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledHeader renders correctly 1`] = `
<h1
  className="sc-bdVaJa jQyXXM"
/>
`;
